build-all.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+BASE_DIR=$(dirname "$0")
+
+cross_build() {
+  SCALA_VER=$1
+  echo "Building with Scala $SCALA_VER"
+  find $BASE_DIR/target/* -type d -exec rm -rf {} \;
+  mvn scala-cross-build:change-version -Pscala-$SCALA_VER
+  mvn install
+}
+
+# ------------------------------------------------
+
+mvn clean
+
+cross_build 2.11
```

Comment:
What if there are compilation errors or test failures during a build for a particular Scala version? 

Should it check the exit code and don't start the next build if the previous one failed?

Reply:
no, no. This shell script is only there for dev convenience.  If I want to quickly run all builds at once locally I run this script. It's not intended for anything else.
On CI a separate build is created per Scala version